phpstan.neon

```diff
@@ -2,5 +2,7 @@ parameters:
     level: 8
     treatPhpDocTypesAsCertain: false
     checkGenericClassInNonGenericObjectType: false
+    bootstrapFiles:
+        - vendor/cakephp/cakephp/src/functions.php
```

Comment:
I am wondering if it would be better to just include `config/bootstrap.php` instead.

Reply:
I've dig a bit more. In fact, CakePHP constants are still a problem with my current proposal because :

- Global functions are defined in `vendor/cakephp/cakephp/src/functions.php`, called by `config/bootstrap.php`
- Global constants are defined in `config/paths.php` called by `config/bootstrap.php`

So, I think you're right.

Reply:
`config/bootstrap.php` will also likely be used to load some dependencies outside composer.